### PR TITLE
Move gever styles

### PR DESCRIPTION
**Pull request by bierik**
Move all gever styles to plonetheme.teamraum.

### CODE REVIEW

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -5,6 +5,12 @@ Changelog
 4.3.1 (unreleased)
 ------------------
 
+- Remove outdated gever styles (moved to plonetheme.teamraum).
+  [Kevin Bieri]
+
+- Expanded/Collapsed triangle style for helptext-trigger.
```
Comment (phgross) [inline docs/HISTORY.txt]: :confused:

**Comment by phgross** [comment]
Looks good :+1:

**Comment by bierik** [comment]
@phgross updated

**Comment by bierik** [comment]
@phgross included ftw.upgrade

**Pull request merged**